zabbix_auto_config/processing.py

```diff
@@ -666,11 +677,27 @@ def do_update(self):
             # if api ver >= 4.4
             if "inventory_mode" in zabbix_host.keys():
                 if zabbix_host["inventory_mode"] != "1":
-                    self.set_inventory(zabbix_host, 1)
+                    self.set_inventory_mode(zabbix_host, 1)
             # else api ver. < 4.4
             else:
                 if zabbix_host["inventory"]["inventory_mode"] != "1":
-                    self.set_inventory(zabbix_host, 1)
+                    self.set_inventory_mode(zabbix_host, 1)
```

Comment:
Wasn't pre 5.0 support completely removed? I think a commit by @mbakke did that? If so: This could be simplified

Reply:
I was not sure about it is completely removed or not. @mbakke Could you confirm it?

Reply:
Indeed, there are other parts of zac that cannot work with < 5.0.  So let's get rid of this compatibility code too.